Source/pack.cpp

```diff
@@ -233,7 +233,7 @@ void UnPackPlayer(PkPlayerStruct *pPack, int pnum, bool netSync)
 	pPlayer->_pGold = SwapLE32(pPack->pGold);
 	pPlayer->_pMaxHPBase = SwapLE32(pPack->pMaxHPBase);
 	pPlayer->_pHPBase = SwapLE32(pPack->pHPBase);
-	pPlayer->_pBaseToBlk = ToBlkTbl[pPlayer->_pClass];
+	pPlayer->_pBaseToBlk = ToBlkTbl[static_cast<std::size_t>(pPlayer->_pClass)];

```

Comment:
Would it make sens for us to move some of this to functions or switches to avoid all the casting?

Reply:
It certainly would, however i consider that separate refactoring. Do you want to do it now, before the enum change?

Reply:
I do to, but if you didn't agree it would make me hesitant to add the casts in the first place :)

Reply:
I believe most of these table-like fetching will become functions later, similar to what I did earlier for max stats.